src/reactviews/pages/QueryResult/table/plugins/cellSelectionModel.plugin.ts

```diff
@@ -58,16 +59,20 @@ export class CellSelectionModel<T extends Slick.SlickData>
             this.selector = this.options.cellRangeSelector;
         } else {
             // this is added by the node requires above
-            this.selector = new CellRangeSelector({
-                selectionCss: {
-                    border: `3px dashed ${tokens.colorStrokeFocus1}`,
+            this.selector = new CellRangeSelector(
+                {
+                    selectionCss: {
+                        border: `3px dashed ${tokens.colorStrokeFocus1}`,
+                    },
                 },
-            });
+                this.webViewState,
+            );
         }
     }
 
-    public init(grid: Slick.Grid<T>) {
+    public async init(grid: Slick.Grid<T>) {
```

Comment:
Making this async without having something that checks for initialization to be complete (awaiting this) can lead to unexpected behavior.  Does this need to be checked somewhere?